src/homeworks/2024_04/test_02/index.ts

```diff
@@ -1 +1,7 @@
-export const reverseWords = () => {};
+const getName = (name: string): string => {
```

Comment:
Suggerisco di chiamare la funzione in un modo descrittivo:

```suggestion
const getReversedWord = (word: string): string => {
```